src/volatile_memory.rs

```diff
@@ -505,13 +525,19 @@ impl Bytes<usize> for VolatileSlice<'_> {
     /// * Write a slice of size 5 at offset 1020 of a 1024-byte `VolatileSlice`.
     ///
     /// ```
-    /// #   use vm_memory::{Bytes, VolatileMemory};
-    /// #   let mut mem = [0u8; 1024];
-    /// #   let mut mem_ref = &mut mem[..];
-    /// #   let vslice = mem_ref.as_volatile_slice();
-    ///     let res = vslice.write(&[1,2,3,4,5], 1020);
-    ///     assert!(res.is_ok());
-    ///     assert_eq!(res.unwrap(), 4);
+    /// # use vm_memory::{Bytes, VolatileMemory};
+    /// #
+    /// # // Create a buffer
+    /// # let mut mem = [0u8; 1024];
+    /// # let mut mem_ref = &mut mem[..];
+    /// #
+    /// # // Get a `VolatileSlice` from the buffer
+    /// # let vslice = mem_ref.as_volatile_slice();
+    /// #
+    /// let res = vslice.write(&[1, 2, 3, 4, 5], 1020);
+    ///
+    /// assert!(res.is_ok());
+    /// assert_eq!(res.unwrap(), 4);
```

Comment:
Looks like the same thing happens here as well :( Feel free to ignore this nit since it's not a change from this PR.

Reply:
Wait, what's happening here? Too many hidden lines?

Reply:
Yep, but it seems fixed now, thanks!